wdl-engine/src/eval/v1/workflow.rs

```diff
@@ -1819,7 +1820,10 @@ workflow test {
         .expect("failed to write WDL source file");
 
         // Analyze the source file
-        let analyzer = Analyzer::new(DiagnosticsConfig::except_all(), |(), _, _, _| async {});
+        let analyzer = Analyzer::new(
+            AnalysisConfig::default().with_diagnostics(DiagnosticsConfig::except_all()),
```

Comment:
Is `except_all()` right here? Would that permit an unrecognized version to be evaluated?